include/mata/nfa/nfa.hh

```diff
@@ -138,7 +133,7 @@ public:
      */
     void unify_final();
 
-    bool is_state(const State &state_to_check) const { return state_to_check < size(); }
+    bool is_state(const State &state_to_check) const { return state_to_check < num_of_states(); }
```

Comment:
if it is a method of delta, then the name is not good, because delta alone does not decide about whether a number is state or not. Maybe contains_state, has_state, ...

Reply:
All three options (the original `is_state()` in the code, `contains_state()` and `has_state()`) sound good to me. Anyone else any preferences?

Reply:
or uses_state

Reply:
That is maybe even better. It clearly explains that the notion of `is a state` has no meaning for delta, and that this function instead checks whether the state is a source or a target for any transition. I would however rename it to `used(State state)` or `used_state(State state)`, or, ideally, `is_state_used(State state)`. What do you think?

Reply:
I added a method `uses_state(State state)` to `Delta`.